coretex/cli/modules/docker.py

```diff
@@ -59,13 +59,15 @@ def start(
     nodeRam: int,
     nodeSwap: int,
     nodeSharedMemory: int,
+    nodeMode: int
 ) -> None:
 
     runCommand = [
         "docker", "run", "-d",
         "--env", f"CTX_API_URL={serverUrl}",
         "--env", f"CTX_STORAGE_PATH={storagePath}",
         "--env", f"CTX_NODE_ACCESS_TOKEN={nodeAccessToken}",
+        "--env", f"CTX_NODE_MODE={nodeMode}",
```

Comment:
Do we need to update the bash script for auto update with this new environment variable?